### PR TITLE
Document loose conda export with --from-history

### PR DESCRIPTION
**Pull request by xhochy**

### CODE REVIEW

**Comment by xhochy** [comment]
Depends on https://github.com/jupyter/repo2docker/pull/864

File: docs/source/howto/export_environment.rst [hunk 1]
```diff
@@ -26,6 +26,24 @@ be resolved (``ResolvePackageNotFound``).
 The solution
 ============
 
+Export your explicit install commands
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you only want to have an ``environment.yml`` that reflects the
```
Comment (betatim) [inline docs/source/howto/export_environment.rst]: How about something like the following? I think we should help unfamiliar users by telling them that they want to do this over other things (unless they know what they are doing, in which case they are probably not reading this).

To get a minimal ``environment.yml`` that only contains the packages you explicitly installed run ``conda env export --from-history environment.yml``. We recommend that you use this option to create your ``environment.yml``. The resulting ``environment.yml`` then contains ...

**Comment by betatim** [comment]
Merged #864, rebase?

File: docs/source/howto/export_environment.rst (hunk 1 above)
Comment (xhochy) [reply docs/source/howto/export_environment.rst]: Thanks, integrated this.

**Comment by xhochy** [comment]
The Travis CI errors seem unrelated.

**Comment by betatim** [comment]
> The Travis CI errors seem unrelated.

Are they ever not unrelated :D ... restarted travis. Seemed like something about running docker containers broke :-/

**Comment by betatim** [comment]
Do you have an idea what the heck travis is trying to tell us? I tried googling for recent changes to how to use docker on travis and such but couldn't find anything. The libraries we install also seem to be the same versions as they were last time the build passed.

**Comment by manics** [comment]
- Broke between 3 and 6 days ago: https://travis-ci.org/github/jupyter/repo2docker/builds?utm_medium=notification&utm_source=github_status
- Nothing in the Travis changelog https://changelog.travis-ci.com/
- I checked the main repo2docker Python dependencies (requests, docker, pytest, websocket-client) but couldn't see any recent releases
- Version of Docker in Travis is unchanged (verified by comparing the build logs for a failed and successful build)

Quite mysterious!

**Comment by betatim** [comment]
```
docker.errors.APIError: 400 Client Error: Bad Request ("OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/usr/local/bin/repo2docker-entrypoint\": permission denied": unknown"
```
keeps coming up as error message. In the GitHub repo the executable bit is set for the file. I wonder if it is owned by "the wrong" user??

**Comment by betatim** [comment]
Travis is happy now. Not sure why it isn't reported as done though. Either way: merging this now.

**Pull request merged**